src/Template/Element/recaptcha.ctp

```diff
@@ -13,6 +13,7 @@ var CaptchaCallback = function() {
     data-theme="<?= $recaptcha['theme'] ?>"
     data-type="<?= $recaptcha['type'] ?>"
     data-size="<?= $recaptcha['size'] ?>"
+    <? if (!empty($recaptcha['callback'])) : ?>data-callback="<?= $recaptcha['callback'] ?>"<? endif; ?>
```

Comment:
Don't use short tags, use `<?php` instead of `<?`.

Reply:
Thanks bro.